### ISSUE TITLE
Webpack Compilation Error v7.3.0

### ISSUE THREAD

**Issue by leungk712** [cypress-io/cypress#16830]
### Current behavior
Hello, I am using Cypress with Vue and SauceLabs and continue to encounter this webpack compilation error whenever I try to run SauceLabs locally:
```
Oops...we found an error preparing this test file:

  cypress\integration\login.spec.js

The error was:

Error: Webpack Compilation Error
./cypress/integration/login.spec.js
Module build failed (from D:/sauce-cypress-runner/7.0.3/bundle/Cache/7.1.0/Cypress/resources/app/packages/server/node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@vue/cli-plugin-babel/preset'

Require stack:
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\config\files\plugins.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\config\files\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\babel-loader\lib\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\loader-runner\lib\loadLoader.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\loader-runner\lib\LoaderRunner.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\NormalModule.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\NormalModuleFactory.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\webpack.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-preprocessor\dist\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-batteries-included-preprocessor\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\lib\plugins\child\index.js
    at handle (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-preprocessor\dist\index.js:172:23)
    at finalCallback (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:257:39)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:306:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:304:22
    at Compiler.emitRecords (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:499:39)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:298:10
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:485:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:482:27
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\neo-async\async.js:2818:7
    at done (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\neo-async\async.js:3522:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies
```
I've followed the node_modules folder and I do see that there is a `@vue/cli-plugin/babel/preset` module seen by the screenshot below

<img width="435" alt="Screen Shot 2021-06-07 at 10 35 08 AM" src="https://user-images.githubusercontent.com/42705941/121057663-bb77e100-c77c-11eb-8b8b-15451d11a16f.png">

And inside the preset.js folder is: 
```
module.exports = require('@vue/babel-preset-app')
```

### Versions
```
    "@babel/runtime": "^7.12.5",
    "babel-core": "^7.0.0-bridge.0",
    "cypress": "^7.3.0",
    "@vue/cli-plugin-babel": "^4.4.6",
    "vue": "^2.6.11"
```

I've deleted my node_modules, uninstalled cypress, reinstalled it, but cannot seem to figure out why it cannot find the module. Any ideas? Do I need to revert Cypress to an earlier version?

**Comment by delphineli** [comment]
we are encountering similar issues in our Jenkins and circle ci jobs.

**Comment by elevatebart** [comment]
Hi @leungk712 @delphineli 

I am trying to find out how this can happen.

The easiest would be to create a reproduction repo that I could clone. 
If it is not possible, can you answer these few questions? That would be tremendously helpful.

What is you package management environment like? Do you dev in a monorepo? 
Are you using yarn? npm? lerna? pnpm? On what versions?
Do you install cypress as a local dependency or rely on the globally installed one?

Thank you in advance

**Comment by delphineli** [comment]
Hi @elevatebart. The project can not be reproduction and clone outside. We use npm and webpack. we have cypress installed in local and for our cd job, we are fetching a fix image. and yes for mono repo. I have fixed the issue by changing test but we still don't understand how the test passed yesterday and without any changes on our side and test failed like this. The changes we made is adding cy.reload() when page load failed. More information for you is we are loading our product in an iframe. This similar problem happened to us twice already. For now, we suspect it's because of the chrome version used by cypress is different. (we can not specify which version of chrome to use for cypress). Last time, we also changed our test to avoid the issue but for me it's not normal that same test code has different effect from day to day. I can share the package.json with you.
`{
  "name": "authoring-ui-react",
  "version": "0.1.0",
  "private": true,
  "url": "https://git.corp.***.com/***",
  "publishConfig": {
    "registry": "https://artifactory.corp.***.com/artifactory/generic-****/"
  },
  "dependencies": {

  },
  "repository": {
    "type": "git",
    "url": "git@git.corp.****.com:***t.git"
  },
  "scripts": {
    "uitest:ci": "CYPRESS_EPOCH=`date +%s` cypress run --headless -b chrome --reporter junit --reporter-options mochaFile=reports/uiresult.xml,toConsole=true",
    "uitest:run": "CYPRESS_EPOCH=`date +%s` cypress run --headless -b chrome --config video=false",
    "uitest:open": "CYPRESS_EPOCH=`date +%s` cypress open",
    "tessa-npm-plugin": "tessa-npm-plugin update",
    "tessa-npm-plugin-pr": "tessa-npm-plugin check --errorOnNewVulnerability"
  },
  "devDependencies": {
    "@babel/polyfill": "7.11.5",
    "@cypress/react": "5.9.1",
    "@svgr/webpack": "5.4.0",
    "@testing-library/jest-dom": "4.1.2",
    "@testing-library/react": "10.4.8",
    "@testing-library/user-event": "7.1.2",
    "@wojtekmaj/enzyme-adapter-react-17": "0.4.1",
    "babel-jest": "26.6.0",
    "babel-loader": "8.1.0",
    "bundlesize": "0.18.0",
    "cypress": "7.5.0",
    "enzyme": "3.11.0",
    "eslint": "7.11.0",
    "http-proxy-middleware": "1.0.4",
    "husky": "4.2.5",
    "jest": "26.6.0",
    "jest-fetch-mock": "3.0.3",
    "launchdarkly-js-client-sdk": "2.18.0",
    "lint-staged": "10.2.9",
    "node-sass": "4.14.1",
    "prettier": "2.0.5",
    "react-dnd-test-backend": "14.0.0",
    "react-dnd-test-utils": "14.0.0",
    "react-scripts": "4.0.2",
    "remotedev": "0.2.9",
    "source-map-explorer": "2.4.2"
  },
  "husky": {
    "hooks": {
      "pre-commit": "lint-staged"
    }
  },
  "lint-staged": {
    "src/**/*.{js,jsx,json,css}": [
      "prettier --single-quote --write",
      "git add"
    ]
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "bundlesize": [
    {
      "path": "./build/static/js/*.chunk.js",
      "maxSize": "1.1mb"
    },
    {
      "path": "./build/static/css/*.chunk.css",
      "maxSize": "140 kB"
    }
  ],
  "browserslist": [
    ">0.2%",
    "not dead",
    "not ie <= 11",
    "not op_mini all"
  ],
  "jest": {
    "resetMocks": false,
    "collectCoverageFrom": [
      "src/**/*.js",
      "!src/beforeSetupTests.js",
      "!src/setupTests.js",
      "!src/intl-enzyme-test-helper.js",
      "!src/user-permissions-test-helper.js",
      "!src/lib/**/*.js"
    ],
    "coverageThreshold": {
      "global": {
        "branches": 81,
        "functions": 91,
        "lines": 91,
        "statements": 91
      }
    },
    "moduleNameMapper": {
      "^dnd-core$": "dnd-core/dist/cjs",
      "^react-dnd$": "react-dnd/dist/cjs",
      "^react-dnd-html5-backend$": "react-dnd-html5-backend/dist/cjs",
      "@ansible/ansible-sdk": "<rootDir>/__mocks__/@ansible/ansible-sdk"
    }
  }
}
`

**Comment by elevatebart** [comment]
@delphineli that is very useful thank you.

The error you are seeing is different from @leungk712 as it relates to cypress end to end testing on a react project.
Would you mind terribly opening another issue? This way we can fix @leungk712 issue with running component testing in Vue CLI and fix yours in parallel on the end to end runner?

Thank you in advance.

**Comment by leungk712** [comment]
Hello @elevatebart,

Unfortunately I cannot clone the repo as it is a work application. I am using npm (v6.14.6), node (v12.18.3). We have Cypress setup locally and we have SauceLabs setup on the Gitlab CI pipeline. Yes to monorepo. Here is our package.json:

`{
  "name": "work-application",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "test:unit": "vue-cli-service test:unit",
    "lint": "vue-cli-service lint",
    "cy:run": "cypress run",
    "cy:run-headless": "cypress run --headless",
    "cy:cache-clear": "cypress cache clear",
    "cy:cache-path": "cypress cache path",
    "cy:install": "cypress install",
    "cy:dependencies-update": "apt-get update -y",
    "cy:dependencies-install": "apt-get -y install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb",
    "cy:verify": "cypress verify"
  },
  "dependencies": {
    "@babel/runtime": "^7.12.5",
    "@jest/globals": "^26.6.2",
    "axios": "^0.19.2",
    "babel-core": "^7.0.0-bridge.0",
    "cleave.js": "^1.6.0",
    "core-js": "^3.7.0",
    "cypress": "^7.1.0",
    "eslint": "^7.13.0",
    "flush-promises": "^1.0.2",
    "fs-extra": "^10.0.0",
    "inject-loader": "^4.0.1",
    "lodash": "^4.17.19",
    "moment": "^2.29.1",
    "pusher-js": "^7.0.1",
    "saucectl": "^0.38.2",
    "sinon": "^9.2.1",
    "uuid": "^8.3.2",
    "vee-validate": "^3.4.4",
    "vue": "^2.6.11",
    "vue-class-component": "^7.2.3",
    "vue-cleave-component": "^2.1.3",
    "vue-property-decorator": "^8.4.2",
    "vue-router": "^3.4.9",
    "vue2-google-maps": "^0.10.7",
    "vuedraggable": "^2.24.3",
    "vuetify": "^2.3.17",
    "vuex": "^3.4.0",
    "vuex-class": "^0.3.2"
  },
  "devDependencies": {
    "@babel/plugin-proposal-object-rest-spread": "^7.12.1",
    "@testing-library/jest-dom": "^5.11.5",
    "@types/jest": "^24.0.19",
    "@types/pusher-js": "^4.2.2",
    "@typescript-eslint/eslint-plugin": "^2.33.0",
    "@typescript-eslint/parser": "^2.33.0",
    "@vue/cli-plugin-babel": "^4.4.6",
    "@vue/cli-plugin-eslint": "~4.4.0",
    "@vue/cli-plugin-router": "~4.4.0",
    "@vue/cli-plugin-typescript": "~4.4.0",
    "@vue/cli-plugin-unit-jest": "~4.4.0",
    "@vue/cli-plugin-vuex": "~4.4.0",
    "@vue/cli-service": "^4.4.6",
    "@vue/eslint-config-standard": "^5.1.2",
    "@vue/eslint-config-typescript": "^5.0.2",
    "@vue/test-utils": "^1.1.1",
    "cucumber": "^6.0.5",
    "eslint-plugin-import": "^2.20.2",
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-standard": "^4.1.0",
    "eslint-plugin-vue": "^6.2.2",
    "node-sass": "^5.0.0",
    "pusher-js-mock": "^0.3.3",
    "sass": "^1.29.0",
    "sass-loader": "^10.1.0",
    "selenium-webdriver": "^4.0.0-alpha.7",
    "typescript": "~3.9.3",
    "vue-cli-plugin-vuetify": "^2.0.6",
    "vue-template-compiler": "^2.6.11",
    "vuetify-loader": "^1.3.0"
  }`
  
  `saucectl run
Running version 0.42.0
11:24:35 INF Reading config file config=.sauce/config.yml
11:24:35 INF Running Cypress in Sauce Labs

                                        (.                          
                                       #.                           
                                       #.                           
                           .####################                    
                         #####////////*******/######                
                       .##///////*****************###/              
                      ,###////*********************###              
                      ####//***********************####             
                       ###/************************###              
                        ######********************###. ##           
                           (########################  ##     ##     
                                   ,######(#*         ##*   (##     
                               /############*          #####        
                           (########(  #########(    ###            
                         .#######,    */  ############              
                      ,##########  %#### , ########*                
                    *### .#######/  ##  / ########                  
                   ###   .###########//###########                  
               ######     ########################                  
             (#(    *#(     #######.    (#######                    
                    ##,    /########    ########                    
                           *########    ########                    

   _____        _    _  _____ ______    _____ _      ____  _    _ _____  
  / ____|  /\  | |  | |/ ____|  ____|  / ____| |    / __ \| |  | |  __ \ 
 | (___   /  \ | |  | | |    | |__    | |    | |   | |  | | |  | | |  | |
  \___ \ / /\ \| |  | | |    |  __|   | |    | |   | |  | | |  | | |  | |
  ____) / ____ \ |__| | |____| |____  | |____| |___| |__| | |__| | |__| |
 |_____/_/    \_\____/ \_____|______|  \_____|______\____/ \____/|_____/
11:24:36 INF Project archived. durationMs=941 size=8266808
11:24:41 INF Project uploaded. durationMs=5166 storageId=[STORAGE ID]
11:24:42 INF Launching workers. concurrency=2
11:24:42 INF Starting suite. region=us-west-1 suite="saucy test"
11:24:45 INF Suite started. browser=chrome platform="Windows 10" suite="saucy test" url=https://app.saucelabs.com/tests/...
11:24:52 INF Suites in progress: 1
11:25:02 INF Suites in progress: 1
11:25:12 INF Suites in progress: 1
11:25:15 ERR Suite finished. passed=false suite="saucy test" url=https://app.saucelabs.com/tests/...
11:25:15 INF console.log output: 
Sauce Cypress Runner 7.0.3

Couldn't determine Mocha version
`

`Oops...we found an error preparing this test file:

  cypress\support\index.js

The error was:

Error: Webpack Compilation Error
./cypress/support/index.js
Module build failed (from D:/sauce-cypress-runner/7.0.3/bundle/Cache/7.1.0/Cypress/resources/app/packages/server/node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@vue/cli-plugin-babel/preset'
Require stack:
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\config\files\plugins.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\config\files\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@babel\core\lib\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\babel-loader\lib\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\loader-runner\lib\loadLoader.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\loader-runner\lib\LoaderRunner.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\NormalModule.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\NormalModuleFactory.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\webpack.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-preprocessor\dist\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-batteries-included-preprocessor\index.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\lib\plugins\child\run_plugins.js
- D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\lib\plugins\child\index.js
    at handle (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\@cypress\webpack-preprocessor\dist\index.js:172:23)
    at finalCallback (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:257:39)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:306:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:304:22
    at Compiler.emitRecords (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:499:39)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:298:10
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:485:14
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\webpack\lib\Compiler.js:482:27
    at D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\neo-async\async.js:2818:7
    at done (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\neo-async\async.js:3522:9)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (D:\sauce-cypress-runner\7.0.3\bundle\Cache\7.1.0\Cypress\resources\app\packages\server\node_modules\tapable\lib\Hook.js:154:20)

This occurred while Cypress was compiling and bundling your test code. This is usually caused by:

- A missing file or dependency
- A syntax error in the file or one of its dependencies

Fix the error in your code and re-run your tests.
`

**Comment by fgcui1204** [comment]
Meet the same issue with u. @leungk712  Have u solved it?

**Comment by leungk712** [comment]
> Meet the same issue with u. @leungk712 Have u solved it?

Hi @fgcui1204, unfortunately have not solved it. Our team has currently put this on hold for now as we're working on other tickets at this time. May circle back to it tomorrow and try again. Good luck.

**Comment by 1beb** [comment]
@elevatebart & @leungk712 I think I have the same error with a reproducible setup (I use docker). Below, I've included my code: 

Dockerfile: 

```
FROM node:lts-alpine

WORKDIR /app

COPY package*.json ./

RUN npm install -f

EXPOSE 8080

CMD ["npm", "run", "serve"]
```

package.json
```
{
  "name": "minitienda",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "test:unit": "vue-cli-service test:unit",
    "test:e2e": "vue-cli-service test:e2e",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "@capacitor/core": "2.4.6",
    "@ionic/vue": "^5.4.0",
    "@ionic/vue-router": "^5.4.0",
    "babel-jest": "^26.6.3",
    "core-js": "^3.6.5",
    "vue": "^3.0.0-0",
    "vue-router": "^4.0.0-0",
    "vue3-mq": "^2.3.3",
    "vuex": "^4.0.0",
    "vuex-persist": "^3.1.3"
  },
  "devDependencies": {
    "@capacitor/cli": "2.4.6",
    "@types/jest": "^24.0.19",
    "@typescript-eslint/eslint-plugin": "^2.33.0",
    "@typescript-eslint/parser": "^2.33.0",
    "@vue/cli-plugin-babel": "~4.5.0",
    "@vue/cli-plugin-e2e-cypress": "~4.5.0",
    "@vue/cli-plugin-eslint": "~4.5.0",
    "@vue/cli-plugin-router": "~4.5.0",
    "@vue/cli-plugin-typescript": "~4.5.0",
    "@vue/cli-plugin-unit-jest": "~4.5.0",
    "@vue/cli-service": "4.5.11",
    "@vue/compiler-sfc": "^3.0.0-0",
    "@vue/eslint-config-typescript": "^5.0.2",
    "@vue/test-utils": "^2.0.0-0",
    "eslint": "^6.7.2",
    "eslint-plugin-vue": "^7.0.0-0",
    "typescript": "~3.9.3",
    "vue-jest": "^5.0.0-0"
  },
  "description": "An Ionic project"
}
```

Docker command: 

```
docker run -it   
  -v $PWD:/e2e   
  -v /tmp/.X11-unix:/tmp/.X11-unix   
  -w /e2e   
  -e DISPLAY   
  --entrypoint cypress   
  cypress/included:3.2.0 open 
  --project .
```

Where DISPLAY an environment variable that is equal to your local IP:0 (192.168.X.X:0). Host system is Ubuntu 21.04. 

As soon as I attempt to run test, I get the error immediately.

**Comment by deyvisrf** [comment]
Any news my friends?
I'm having the same problem with my repository.

https://github.com/deyvisrf/QAChallenge/tree/saucelabs-testes

**Comment by JessicaSachs** [comment]
These are unrelated issues... The original issue is about a compilation failure. When working with compilation issues, we need a reproduction that includes the package.json, an example of a spec file, and the Cypress setup code (plugins file and support files).

Essentially, we need to know what you're trying to compile inside of your spec files.

Going to close this unless I get a reproduction.

**Comment by deyvisrf** [comment]
@JessicaSachs all files found are in this repository: https://github.com/deyvisrf/QAChallenge/tree/saucelabs-testes
